pom.xml

```diff
@@ -48,13 +49,14 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-      <dependency>
-        <groupId>io.jenkins.plugins</groupId>
-        <artifactId>snakeyaml-api</artifactId>
-      </dependency>
     </dependencies>
   </dependencyManagement>
+
   <dependencies>
+    <dependency>
```

Comment:
Can be removed once we migrate to Java 21 (see TODO in hudson.plugins.locale.PluginImpl#parse)